riff-raff/app/persistence/RestrictionConfigDynamoRepository.scala

```diff
@@ -0,0 +1,22 @@
+package persistence
+
+import java.util.UUID
+
+import com.gu.scanamo.Table
+import restrictions.{RestrictionConfig, RestrictionsConfigRepository}
+
+object RestrictionConfigDynamoRepository extends DynamoRepository with RestrictionsConfigRepository {
```

Comment:
I hate this kind of redundant interface, but the alternative isn't as simple as it should be without lots of wider changes.